requirements/base.txt

```diff
@@ -1,10 +1,12 @@
+boto==2.22.1                        # MIT
 Django==1.7.5 						# BSD License
-Markdown==2.6    					# BSD
 django-model-utils==2.2 			# BSD
 djangorestframework==2.4.4			# BSD
-ipython==2.4.1						# BSD
 django-rest-swagger==0.2.8  		# BSD
 djangorestframework-csv==1.3.3		# BSD
 django-countries==3.2               # MIT
-elasticsearch-dsl==0.0.9           # Apache 2.0
+elasticsearch-dsl==0.0.9            # Apache 2.0
+ipython==2.4.1						# BSD
```

Comment:
That does seem out of place--not sure how that made it in there.  I'll remove it.